includes/kaltura_derivatives.inc

```diff
@@ -68,10 +68,21 @@ function islandora_video_streaming_kaltura_upload($object, $force = false) {
 
       $entity = kaltura_entry_create(array('kaltura_entryid' => $entry->id));
       kaltura_map_properties($entry, $entity);
-
-      kaltura_save_entry_metadata($entity);
+      $entity->kaltura_title = $object->id; 
```

Comment:
If the kaltura_title and kaltura_description were assigned before `kaltura_map_properties()` would that function and `kaltura_entry_save()` do the work of the `try` with the `KalturaBaseEntry()` we have added?

Reply:
According to variable information shown by xdebug, if  kaltura_title and kaltura_description were assigned before, the values previously assigned to kaltura_title and kaltura_description are overwritten by  `kaltura_map_properties()`. In addition, using the two statements assigned after `kaltura_map_properties()` and using `kaltura_save_entry_metadata()` does not work.